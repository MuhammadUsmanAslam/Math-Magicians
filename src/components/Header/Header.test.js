import renderer from "react-test-renderer";
import Header from "./Header";
import { BrowserRouter as Router } from "react-router-dom";

test("render header component correctly", () => {
	const tree = renderer
		.create(
			<Router>
				<Header></Header>
			</Router>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
