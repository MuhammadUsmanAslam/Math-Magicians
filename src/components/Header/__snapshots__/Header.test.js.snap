// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header component 1`] = `
<div
  className="header"
>
  <a
    className="header__logo"
    href="/"
    onClick={[Function]}
  >
    Math Magicians
  </a>
  <ul
    className="header__nav-links"
  >
    <li>
      <a
        className=""
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className=""
        href="/quotes"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;

exports[`render header component correctly 1`] = `
<div
  className="header"
>
  <a
    className="header__logo"
    href="/"
    onClick={[Function]}
  >
    Math Magicians
  </a>
  <ul
    className="header__nav-links"
  >
    <li>
      <a
        className=""
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className=""
        href="/quotes"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
